src/index.ts

```diff
@@ -50,6 +51,12 @@ const options = {
         default: false,
         description: 'Should specifiers be sorted?',
     },
+    importOrderPreventSortingSideEffectNodes: {
```

Comment:
Maybe something like this?
```suggestion
    importOrderSideEffects: {
```

Reply:
renamed